client/commonFramework/execute-challenge-stream.js

```diff
@@ -18,7 +18,7 @@ window.common = (function(global) {
     const originalCode = code;
     const head = common.arrayToNewLineString(common.head);
     const tail = common.arrayToNewLineString(common.tail);
-    const combinedCode = head + code + tail;
+    const combinedCode = head + code + '\n' + tail;
```

Comment:
I was thinking the same before I pushed it up. We can definitely add it as a precaution!

I hesitated to because most of the "head" code I looked through does not end with comments (and if so, most of them put a newline after their code). But if there were ever a scenario that did, it would cut off the first line of of the "code" from the editor.

I'll go ahead and throw it in as well as a precaution.